blog/en/blog/2025/08/31/2025-august-monthly-report.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "2025 Monthly Report (August 01 - August 31)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.api7.ai/uploads/2025/08/29/dfR8DC8m_aug-monthly-report-cover-en.webp
+---
+
+> Recently, we've introduced and updated some new features, including adding ai-aliyun-content-moderation plugin and allowing environment variables in openid-connect plugin, etc. For more details, please read this month's newsletter.
+
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way and is made possible by the collective effort of our community.
+
+From August 1st to August 31, 16 contributors made 56 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.api7.ai/uploads/2025/08/29/SWsZprNc_aug-contributor-list.webp)
+
+![New Contributors List](https://static.api7.ai/uploads/2025/08/29/7SSxLwiC_aug-new-contributors.webp)
+
+## Feature Highlights
+
+### 1. Add ai-aliyun-content-moderation Plugin
+
+PR: https://github.com/apache/apisix/pull/12530
+
+Contributor: [Revolyssup](https://github.com/Revolyssup)
+
+This PR:
+
+1. refactors and improves the chunk decoding from the LLM backend into a separate sse.lua.
+
+2. introduces `lua_response_filter` to run `lua_body_filter` as defined by each plugin, because `lua_body_filter` can't be processed directly in AI responses, which are managed by APISIX.
```

Comment:
Why use this format? Just write a paragraph.

The title and description have mismatches. The title only mentions the addition of one plugin, while the description mentions other plugins as well. This PR is quite large - please confirm with @Revolyssup